src/lib.rs

```diff
@@ -157,4 +209,33 @@ mod tests {
         assert!(output.is_ok());
         assert_eq!(output.unwrap(), "\nstruct Test\n{};\n");
     }
+
+    #[test]
+    fn format_custom() {
+        let input = r#"
+            struct Test {
+                bool field;
+            };
+        "#;
+        {
+            let output = clang_format_with_style(
+                input,
+                &ClangFormatStyle::Custom("{ BasedOnStyle: Mozilla, IndentWidth: 8 }"),
+            );
+            assert!(output.is_ok());
+            assert_eq!(
+                output.unwrap(),
+                "\nstruct Test\n{\n        bool field;\n};\n"
+            );
+        }
+
+        {
+            let output = clang_format_with_style(
+                input,
+                &ClangFormatStyle::Custom("{ BasedOnStyle: Mozilla, IndentWidth: 4 }"),
```

Comment:
Please make this a multi-line string and find some style argument that accepts quotes to see if those work here.